src/tasks/AutoBidAuctionTask.py

```
@@ -0,0 +1,425 @@
+                    raise WaitFailedException("匹配阶段重试次数超出上限")
+
+    def _stage_confirm(self, box_confirm, re_confirm):
+        self._check_stop()
+        self.log_info("等待确认")
+        self.wait_ocr(
+            box=box_confirm,
+            match=re_confirm,
+            time_out=5,
+            target_height=1080
+        )
+        self.log_info("确认按钮已出现，正在点击...")
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)